README.md

```diff
@@ -21,35 +21,74 @@ This role requires root access, so either configure it in your inventory files,
 > playbook.yml:
 ```YAML
 - hosts: dnsserver
+  become: yes
   roles:
     - role: aalaesar.install_nextcloud
-      become: yes
 ```
 
 ## Role Variables
 
 Role's variables (and their default values):
 
-### Installation configuration
-> Source location will be calculated following channel, version and branch values.
+### Choose the version
```

Comment:
Yeah, I also thought about this, this afternoon.
I'll do simple examples just below the vars declarations, then add some variation to the final cases..